pkgs/desktops/gnome-3/3.22/default.nix

```diff
@@ -238,6 +238,8 @@ let
 
   vte_290 = callPackage ./core/vte/2.90.nix { };
 
+  vte_291 = callPackage ./core/vte/2.91.nix { };
```

Comment:
If there are not breaking changes. I would update `vte_290` and rename it to `vte_29` rather then having a dedicated package.